src/site/content/en/blog/state-of-css-2022/index.md

```diff
@@ -422,19 +420,16 @@ are many exciting and meaningful design decisions to make when using
 
 #### Resources
 
-- Spec:
-  [https:/www.w3.org/TR/css-contain-3/#container-queries](https://www.w3.org/TR/css-contain-3/#container-queries)   
-- Explainer:
-  [https://css.oddbird.net/rwd/query/explainer/](https://css.oddbird.net/rwd/query/explainer/)   
-- MDN:
-  [https://developer.mozilla.org/docs/Web/CSS/CSS_Container_Queries](https://developer.mozilla.org/docs/Web/CSS/CSS_Container_Queries)   
+- [Spec](https://www.w3.org/TR/css-contain-3/#container-queries)
+- [Explainer](https://css.oddbird.net/rwd/query/explainer/)
+- [MDN](https://developer.mozilla.org/docs/Web/CSS/CSS_Container_Queries)
 - The new responsive:
-  [/new-responsive/#responsive-to-the-container](/new-responsive/#responsive-to-the-container)  
+  [/new-responsive/#responsive-to-the-container](/new-responsive/#responsive-to-the-container)
```

Comment:
This particular entry looks kind of weird? And honestly the rest are a bit strange too, since they're a mix of text following by `<a>` tags and some `<a>` tags that include the text as the link.

![Screen Shot 2022-06-01 at 3 47 46 PM](https://user-images.githubusercontent.com/1749548/171489347-e8a2782a-39a3-4ee4-b80f-b2b366c6d689.png)